src/main/java/edu/oregonstate/cs361/battleship/Main.java

```diff
@@ -250,8 +250,8 @@ public static BattleshipModelUpdated placeShipUpdatedEasy(BattleshipModelUpdated
             Ship[] CArray = model.resetArrayUpdated(model, false);
             int computerLength = model.getShipByID(CArray[id].getName()).getLength() - 1;
 
-            model.getShipByID(CArray[id].getName()).setStart(id, id);
-            model.getShipByID(CArray[id].getName()).setEnd(id + computerLength, id);
+            model.getShipByID(CArray[id].getName()).setStart(id + 1, id + 1);
```

Comment:
Otherwise the first ship (aircraft carrier) starts at 0,0 to 0,4 (aka off the board)